packages/cf.js/test/provider.spec.ts

```diff
@@ -10,12 +11,22 @@ import {
   RejectInstallEventData
 } from "../src/types";
 
-import { TEST_APP_INSTANCE_INFO, TestNodeProvider } from "./fixture";
+import { TestNodeProvider } from "./fixture";
 
-describe("CF.js Provider", async () => {
+describe("CF.js Provider", () => {
   let nodeProvider: TestNodeProvider;
   let provider: Provider;
 
+  const TEST_APP_INSTANCE_INFO: AppInstanceInfo = {
+    id: "TEST_ID",
+    asset: { assetType: AssetType.ETH },
+    abiEncodings: { actionEncoding: "uint256", stateEncoding: "uint256" },
+    appId: "0x1515151515151515151515151515151515151515",
+    myDeposit: new BigNumber("0"),
```

Comment:
@mvanderh Maybe we can define a constant `BIG_NUMBER_ZERO` for this?

Reply:
Already exists in ethers.constants. Will refactor